src/store/modules/requests/mutations.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  addRequest(state, payload) {
+    state.requests.push(payload);
+  }
+}
```

Comment:
@HVRolim newline no final do arquivo
```suggestion
}

```